package.json

```diff
@@ -8,7 +8,7 @@
     "url": "https://git.heroku.com/ft-six-degrees-be-app.git"
   },
   "engines": {
-    "node": "6.9.1"
+    "node": "6.11.1"
```

Comment:
Might be worth doing "^6.11.1" to ensure future security patches get picked up too.